src/classes/CRLP_RollupContact_TEST.cls

```diff
@@ -496,6 +511,7 @@ public class CRLP_RollupContact_TEST {
         System.assertEquals('New', c.Department);
 
         System.assertEquals(1, c.npo02__OppsClosed2YearsAgo__c, 'The number of close lost opps should be 1');
+        System.assertEquals(wonCount, c.npo02__OppsClosedLastYear__c, 'The number of closed won opps should be equal to wonCount');
```

Comment:
The assert should show the value of the var it should equal as opposed to just the var name.
```suggestion
        System.assertEquals(wonCount, c.npo02__OppsClosedLastYear__c, 'The number of closed won opps should be equal to ' + wonCount);
```